src/voices/voices2.rs

```diff
@@ -0,0 +1,52 @@
+use rdev::Key;
+
+fn get_ordinal(key: Key) -> u8 {
+    let ptr_to_option = (&key as *const Key) as *const u8;
```

Comment:
The underlying representation for an enum with under 256 variants starts with an ordinal byte, guaranteed. Get that number!